docs/odata-data-aggregation-ext/odata-data-aggregation-ext.md

```diff
@@ -1276,7 +1276,7 @@ results in
 
 ##### <a name="StandardAggregationMethodcountdistinct" href="#StandardAggregationMethodcountdistinct">3.2.1.3.5 Standard Aggregation Method `countdistinct`</a>
```

Comment:
The [`name` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#name) of the anchor element is deprecated, let's try

```suggestion
##### <span id="StandardAggregationMethodcountdistinct" href="#StandardAggregationMethodcountdistinct">3.2.1.3.5 Standard Aggregation Method `countdistinct`</span>
```

Reply:
`<a id="..." href="...">` if we want to keep the self-link.